go/vt/vtctl/reparentutil/emergency_reparenter.go

```diff
@@ -594,10 +606,10 @@ func (erp *EmergencyReparenter) reparentReplicas(
 
 	primaryErr := handlePrimary(topoproto.TabletAliasString(newPrimaryTablet.Alias), newPrimaryTablet)
 	if primaryErr != nil {
-		erp.logger.Warningf("primary failed to PopulateReparentJournal")
+		erp.logger.Warningf("primary failed to promote the new primary")
 		replCancel()
 
-		return nil, vterrors.Wrapf(primaryErr, "failed to PopulateReparentJournal on primary: %v", primaryErr)
+		return nil, vterrors.Wrapf(primaryErr, "failed to promote the new primary: %v", primaryErr)
```

Comment:
We log a warning, and also return an error? Why do we log it as a warning and not log it at error level?
Please review this error message as well to see whether it makes sense to append the tablet alias to it.

Reply:
Fixed. We log an error via the logger. The logger might choose to ignore it too, so the error is required to be sent back.